micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspectSpec.groovy

```diff
@@ -9,23 +9,28 @@ class MetricsAspectSpec extends Specification {
 
     def 'should turn "#url" into valid metric name: "#expectedMetricName"'() {
         when:
-            def metricName = MetricsAspect.metricName(url)
+        def metricName = MetricsAspect.metricName(url)
```

Comment:
In other specs we intend statements after labels, so it'd be nice to keep it the same also in this spec.

You can tell Intellij to format specs according to this style. You can find the instructions here: https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13